### ISSUE TITLE
JJW Design Map not working - Google API error - Googlecode dismissed

### ISSUE THREAD

**Issue by mayerelyashiv** [salesagility/SuiteCRM#1438]
Hi,
i found in an update here: stackoverflow.com/questions/37087583/goo...not-working-in-https
that since Google on May the 12th 2016 decommissioned googlecode

modules/jjwg_Maps/views/view.map_markers.php is not working anymore.

Solution proposed:
1 to download scripts from here:
github.com/sytelus/CryptoJS
2 to upload them here: modules/jjwg_Maps/javascript/crypto
3 to modify
modules/jjwg_Maps/views/view.map_markers.php 
changing:
```
<script type="text/javascript" src="//crypto-js.googlecode.com/svn/tags/3.1.2/build/rollups/sha3.js"></script>
<script type="text/javascript" src="//crypto-js.googlecode.com/svn/tags/3.1.2/build/components/enc-base64-min.js"></script>
```

in:
```
<script type="text/javascript" src="modules/jjwg_Maps/javascript/crypto/rollups/sha3.js"></script>
<script type="text/javascript" src="modules/jjwg_Maps/javascript/crypto/components/enc-base64-min.js"></script>
```
and
changing:
```
MarkerClusterer.IMAGE_PATH = "//google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclustererplus/images/m";
```
in
```
MarkerClusterer.IMAGE_PATH = "//cdn.rawgit.com/googlemaps/js-marker-clusterer/gh-pages/images/m";
```

**Comment by mayerelyashiv** [comment]
Hi, i found that on a clean 7.6.3 istance it works.
It's an upgrade bug.

I posted a better solution here:
https://suitecrm.com/forum/installation-upgrade-help/9471-jjw-design-map-not-working-on-7-6-3-after-upgrade#32473

**Comment by PeterTheOne** [comment]
#1422 tries to fix this..

**Comment by mayerelyashiv** [comment]
Thank you Peter

**Comment by shogunpol** [comment]
@mayerelyashiv  
Can you confirm is this issue still exist in recent version of SuiteCRM(7.7.1), ive checked it, and seems to work correctly.

**Comment by mayerelyashiv** [comment]
Hi shogunpol, thanks i'll report here as i test it. Btw it was fixed on 7.6.3

**Comment by ijdavie** [comment]
HI @mayerelyashiv, 

I will close this currently as a fix was put in for 7.6.3, if the problem reappears it can be reopened.

**Comment by mayerelyashiv** [comment]
Yes, thank you

2016-09-01 16:04 GMT+02:00 Ian Davie <notifications@github.com>:

> HI @mayerelyashiv
> <https://wmphighrise.appspot.com/r/7ab48605fcb587f837178836d02d765c?d=https%3A%2F%2Fgithub.com%2Fmayerelyashiv>,
>
>
> I will close this currently as a fix was put in for 7.6.3, if the problem
> reappears it can be reopened.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://wmphighrise.appspot.com/r/7ab48605fcb587f837178836d02d765c?d=https%3A%2F%2Fgithub.com%2Fsalesagility%2FSuiteCRM%2Fissues%2F1438%23issuecomment-244088975>,
> or mute the thread
> <https://wmphighrise.appspot.com/r/7ab48605fcb587f837178836d02d765c?d=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FACxm7ZbI7PoIZdEVls4jJYH4nkZbGbjEks5qltuAgaJpZM4IiB19>
> .
>



-- 
*Mayer Elyashiv*
Lion Solution Srls
Via della Provvidenza, 37/A - 35030 Rubano (PD)
T. 049-635764
P.IVA: 04937450288
www.lionsolution.it
<https://wmphighrise.appspot.com/r/7ab48605fcb587f837178836d02d765c?d=http%3A%2F%2Fwww.lionsolution.it%2F>
 | mayer@lionsolution.it
 *http://it.linkedin.com/in/mayerelyashiv
<https://wmphighrise.appspot.com/r/7ab48605fcb587f837178836d02d765c?d=http%3A%2F%2Fit.linkedin.com%2Fin%2Fmayerelyashiv>*
